<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="imageViewer_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 238, 238);
font: 75 10pt &quot;Adobe 고딕 Std B&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageViewer_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 238, 238);
font: 75 10pt &quot;Adobe 고딕 Std B&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageViewer_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>  
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 238, 238);
font: 75 10pt &quot;Adobe 고딕 Std B&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
    <widget class="QLabel" name="imageViewer_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
    <rect>
        <x>660</x>
        <y>420</y>
        <width>81</width>
        <height>21</height>
    </rect>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 238, 238);
        font: 75 10pt &quot;Adobe 고딕 Std B&quot;;</string>
    </property>
    <property name="text">
    <string>4</string>
    </property>
 </widget>
 </widget>
<resources/>
<connections/>
</ui>